tests/run.sh

```diff
@@ -94,6 +94,8 @@ function modify_services_for_testing() {
     sed -e 's/DOWNLOAD_SPEED/#DOWNLOAD_SPEED/' -i ../docker-compose.testing.yml
     # whitelist *all* IPs/DNS names in the gateway (will be accessed via dynamically assigned IP in tests)
     sed -e 's/ALLOWED_HOSTS: .*/ALLOWED_HOSTS: ~./' -i ../docker-compose.testing.yml
+    # make sure that MinIO healthcheck from docker-compose.demo.yml is used independently from compose files sequence
+    sed -e '/^[[:space:]]*healthcheck\|test\|interval\|timeout\|retries\|start_period/d' -i ../docker-compose.storage.minio.yml
```

Comment:
Although I see your reasoning (good commit message, by the way!) it is very inconvenient for local development to edit a file that is included in git. After running a test this modification will be on the way for my local git status.

As you (kind of) said in the commit message, both in `demo` script and the test code we are placing the demo (or testing) yml file after `docker-compose.storage.minio.yml`, and these should be the only external entry points for users, tools, ci, etc.

`docker-compose.demo.yml` overrides other keys for other services in `docker-compose.yml` so it is already expected to be the last file added into the composition.

So please drop this commit :)

I personally don't think it is necessary, but one thing we could do for future reference is to add a comment in [docker_compose_manager.py](https://github.com/mendersoftware/integration/blob/b3a0ad7e219bcf9d16896b9ceb5fa7baf4bf5553/testutils/infra/container_manager/docker_compose_manager.py#L33-L34) to explicitly say that the order there is important.